recipes/loomxpy/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "loomxpy" %}
+{% set version = "0.4.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/aertslab/LoomXpy/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: c5c3c29e3abde34acb38ff034225ee849e67cfe84a89b81b8ce6f90fd694ccb6
+
+build:
+  number: 0
+  noarch: python
+  run_exports: {{ pin_compatible("loomxpy", max_pin="x.x") }}
+
+requirements:
+  build:
+    - python >=3.7.1
```

Comment:
Can you please use python_min here https://github.com/bioconda/bioconda-recipes/blob/195a5ef45325b5232d7f9fc3eeeae942df22b920/recipes/plateo/meta.yaml#L25 below and also in the test ... as in the plateo example